docs/sources/developers/plugins/plugin.schema.json

```diff
@@ -471,6 +471,31 @@
           "default": false
         }
       }
+    },
+    "extensions": {
+      "type": "object",
+      "description": "Extends various parts of the Grafana UI with commands or links.",
+      "additionalProperties": false,
+      "properties": {
+        "links": {
+          "type": "array",
+          "description": "Expose a page link that can be used by Grafana core or other plugins to navigate users to the plugin",
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string"
+              },
```

Comment:
is the id where the link is supposed to be used? (like `panel-menu`)  ?

Reply:
No, its the Id of the link. You can then choose to place it in e.g. the panel-menu. Please read the design doc linked in the epic for more details.